src/java.desktop/share/classes/javax/swing/border/CompoundBorder.java

```diff
@@ -137,6 +137,8 @@ public void paintBorder(Component c, Graphics g, int x, int y, int width, int he
      * Reinitialize the insets parameter with this Border's current Insets.
```

Comment:
```suggestion
     * Reinitializes the insets parameter with this Border's current Insets.
```
The same: the verb at start of a javadoc is in third person singular.